cmd/boost/run.go

```diff
@@ -22,6 +28,39 @@ var runCmd = &cli.Command{
 	Usage:  "Start a boost process",
 	Before: before,
 	Action: func(cctx *cli.Context) error {
+		if cpuprofile := cctx.String("pprof.cpu"); cpuprofile != "" {
+			profile, err := os.Create(cpuprofile)
+			if err != nil {
+				return err
+			}
+
+			if err := pprof.StartCPUProfile(profile); err != nil {
+				return err
+			}
+			defer pprof.StopCPUProfile()
+		}
+
+		if memprofile := cctx.String("pprof.mem"); memprofile != "" {
+			f, err := os.Create(memprofile)
+			if err != nil {
+				return err
+			}
+			defer f.Close()
+			runtime.GC() // get up-to-date statistics
+			if err := pprof.WriteHeapProfile(f); err != nil {
```

Comment:
I think this will write a heap profile immediately, which may not be that useful. Probably we want to see a heap profile at a particular point in time, which we can get through the pprof web service.

Maybe we don't need the pprof.cpu and pprof.mem flags at all and we can just rely on the pprof web service?

Reply:
I agree, maybe I went too far with the cpu and mem profiles. Let's just have the web service for now, and add more later if we need it.